sdk/javascript/src/index.js

```diff
@@ -1,18 +1,14 @@
-let baseUrl = ''
-if (process.env.NODE_ENV === 'production') {
-  baseUrl = 'https://api.attractify.io/v1'
-} else {
-  baseUrl = 'http://127.0.0.1:8080/v1'
-}
-
 import Fifo from 'localstorage-fifo'
 
+
 class Attractify {
-  constructor(authToken) {
+  constructor(authToken, configOptions) {
+
     this.state = { isIdentified: false }
     this.authToken = authToken
     this.context = null
     this.queue = new Fifo({ namespace: 'trackings' })
+    this.baseUrl = configOptions.apiUrl
```

Comment:
There should be a check, that a configOptions object was really passed and that it contains an API URL. Please throw an exception otherwise.